tests/tools/test_devservices_healthcheck.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+from tools.devservices_healthcheck import run_cmd
+
+
+def test_cmd_run_fail():
+    with pytest.raises(SystemExit) as se:
+        run_cmd(["i_dont_exist"], retries=1)
```

Comment:
the more important test is something which exits nonzero, such as `sys.executable, '-c', 'raise SystemExit(1)` -- conventionally the `pytest.raises` assignment is usually `as excinfo`